### ISSUE TITLE
Search not working

### ISSUE THREAD

**Issue by bosborne** [gdg-managua/jekyll-mdl#29]
I went to http://www.gdgmanagua.org/jekyll-mdl/, entered "Bacon" in the search field, hit return (or clicked the magnifying glass), no result, nothing happens. Is a search result supposed to appear? The search also does not work with the code installed on my laptop.

**Comment by oscarmcm** [comment]
Here is a local screenshot, that show the search working. The search term that I use is _goog_

![image](https://cloud.githubusercontent.com/assets/2553459/11461428/b183d9dc-96c8-11e5-9fb8-b002429c16dd.png)

@bosborne Sorry, the demo is not updated with the last changes in master. I will update the demo in a while. Let me know if the problem persists :smile:

**Comment by bosborne** [comment]
Oscar, thank you for your reply. Does this search rely on Google? Is that why the search does not work when the Web server is running on my laptop?

**Comment by oscarmcm** [comment]
@bosborne Nope, the search is based in the _feed.xml_ file, when you put more than 3 words in the search, then find if any post in the file contains this word. You have some entries written in your local setup?

**Comment by bosborne** [comment]
Yes, have some entries, and the match works about 1 in 3 or 4 attempts. The requirement for having more than 2 words to get a match is odd, many times, or most of the time, people enter 1 word, expect to see the matches. Anyway, does not seem to work reliably.
